index.html

```diff
@@ -0,0 +1,686 @@
+                        <h3 class="latest-news-item-h3"><a class="latest-news-item-h3-a" href="#">No clue how to handle coronavirus as a startup founder? Here's what we're doing</a></h3>
+                        <h4 class="latest-news-item-h4"><a class="latest-news-item-h4-a" href="#">Patrick de Laive</a> - 4 hours ago</h4>
+                    </div>
+                </div>
+                <div class="latest-news-item">
+                    <img class="latest-news-item-img" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Untitled-design21-516x270.png" alt="item 1">
+                    <div class="latest-news-item-text-container">
+                        <h3 class="latest-news-item-h3"><a class="latest-news-item-h3-a" href="#">No clue how to handle coronavirus as a startup founder? Here's what we're doing</a></h3>
+                        <h4 class="latest-news-item-h4"><a class="latest-news-item-h4-a" href="#">Patrick de Laive</a> - 4 hours ago</h4>
+                    </div>
+                </div>
+                <div class="latest-news-item">
+                    <img class="latest-news-item-img" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Untitled-design21-516x270.png" alt="item 1">
+                    <div class="latest-news-item-text-container">
+                        <h3 class="latest-news-item-h3"><a class="latest-news-item-h3-a" href="#">No clue how to handle coronavirus as a startup founder? Here's what we're doing</a></h3>
+                        <h4 class="latest-news-item-h4"><a class="latest-news-item-h4-a" href="#">Patrick de Laive</a> - 4 hours ago</h4>
+                    </div>
+                </div>
+                <div class="latest-news-item">
+                    <img class="latest-news-item-img" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Untitled-design21-516x270.png" alt="item 1">
+                    <div class="latest-news-item-text-container">
+                        <h3 class="latest-news-item-h3"><a class="latest-news-item-h3-a" href="#">No clue how to handle coronavirus as a startup founder? Here's what we're doing</a></h3>
+                        <h4 class="latest-news-item-h4"><a class="latest-news-item-h4-a" href="#">Patrick de Laive</a> - 4 hours ago</h4>
+                    </div>
+                </div>
+                <div class="latest-news-item">
+                </div>
+            </div>
+        </div>
+
+        <div class="general-container">
+                <div class="funding-header">
+                    <h1 class="section-title"><a class="section-title-a" href="#"> Latest funding rounds on Index.co ></a></h1>
+                    <div class="funding-navbar">
+                        <a class="funding-navbar-a" href="#">Add info</a>
+                        <a class="funding-navbar-a" href="#">Get updates by email</a>
+                        <a class="funding-navbar-a last-nav-item" href="#">See more</a>
```

Comment:
- [ ] This section also displays differently around 990px. It should have 3 elements and occupy 100% width but it's showing this:
![Screenshot from 2020-03-18 16-30-00](https://user-images.githubusercontent.com/51926937/77013461-bb03e800-6935-11ea-8bee-07a8a091c08f.png)

The original:
![Screenshot from 2020-03-18 16-30-00](https://user-images.githubusercontent.com/51926937/77013485-c525e680-6935-11ea-8ced-14e370de9f0b.png)
![Screenshot from 2020-03-18 16-30-19](https://user-images.githubusercontent.com/51926937/77013488-c5be7d00-6935-11ea-8cd5-bf7cebb38739.png)

- [ ] There's a moment when this section has two articles in one row, that never happens on the original page. Please fix it.
![Screenshot from 2020-03-18 16-31-00](https://user-images.githubusercontent.com/51926937/77013593-f43c5800-6935-11ea-8d80-ca91578a9025.png)

- [ ] In the mobile and desktop version the image it's too big. Please adjust it.
![Screenshot from 2020-03-18 16-32-02](https://user-images.githubusercontent.com/51926937/77013643-0fa76300-6936-11ea-9572-4929b6560195.png)

![Screenshot from 2020-03-18 16-33-09](https://user-images.githubusercontent.com/51926937/77013698-2fd72200-6936-11ea-8a87-fc6001750ebf.png)